model/task_lifecycle_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	. "github.com/smartystreets/goconvey/convey"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	mgobson "gopkg.in/mgo.v2/bson"
```

Comment:
I have a recollection we were trying to avoid including mgo to not make it harder when we eventually tear it out

Reply:
Is there something that I should rather use instead? Or is it better to hardcode it?

Reply:
I don't know of something else to generate it.
In cases where we've really wanted an ID we've inserted it into the db without an ID and gotten it back out of the db (the driver/db (depending on the language) will assign an objectID).
This was always a Sam thing...

But do we need a real object ID? Potentially the version's ID could be "v1" or something

Reply:
I had the patchID hardcoded before,  I can switch it back to that. 
Getting the version Ids, the patchIds, and the issue numbers to line up for CLI patches are very complicated. I wasn't able to get it to work with the version ID being "v1". 
I can't insert the into the database with no ID  and then use p.Id because there is a circular dependency between version and patch, each relying on each other's Id. 
I tried many different variations with the version id, patch id, and issue for the CommitQueue item, and this was the only thing that ended up working. If you can think of any other ideas that you think would be better, please let me know.